vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/account_selection_rule.rb

```diff
@@ -0,0 +1,19 @@
+module SecureRooms
+
+  module AccessRules
+
+    class AccountSelectionRule < BaseRule
+
+      def self.condition(_user, _card_reader, accounts, selected)
+        if selected.present?
+          Verdict.new(:grant)
+        elsif accounts.present? && selected.blank?
+          Verdict.new(:pending, "Must select Account")
+        end
```

Comment:
Is there a possibility of this not hitting either of the conditions? Does this need to return a default `pass` verdict?